cheatsheets/Java_Security_Cheat_Sheet.md

```diff
@@ -598,20 +602,24 @@ class AesGcmSimple {
 
 </details>
 
-#### Symmetric example using built in JCA/JCE classes
+_**End of code snippet.**_
+
+#### Symmetric example using built-in JCA/JCE classes
 
-If you absolutely cannot use a separate library, it is still possible to use the built JCA/JCE classes but it is strongly recommended to have a cryptography expert review the full design and code.
+If you absolutely cannot use a separate library, it is still possible to use the built JCA/JCE classes but it is strongly recommended to have a cryptography expert review the full design and code, as even the most trivial error can severely weaken your encryption.
 
 The following code snippet shows an example of using AES-GCM to perform encryption/decryption of data.
 
-A few contraints/pitfalls with this code:
+A few constraints/pitfalls with this code:
 
 - It does not take into account key rotation or management which is a whole topic in itself.

```

Comment:
I think this would be a good place for a reference to that link you had referenced earlier in your response to one of my comments: 
https://crypto.stackexchange.com/questions/41601/aes-gcm-recommended-iv-size-why-12-bytes/41610#41610

In particular, the 2nd answer is very relevant here and I think that fits well with this bullet. It's way less obtuse than most of the relevant NIST special publications.

Reply:
Addressed in #1160